buildenv/docker/mkdocker.sh

```diff
@@ -124,6 +131,15 @@ parse_options() {
         ;;
     esac
   done
+
+  # If --build was specified without an engine, and `docker` isn't on $PATH,
+  # and `podman` is on $PATH, then assume they're okay to use `podman`.
+  if [ "${action}" = "build" ] && \
+     [ "${engine_specified}" -eq 0 ] && \
+     ! command -v "$engine" >/dev/null 2>&1 && \
+     command -v "podman" >/dev/null 2>&1; then
+    engine="podman"
+  fi
```

Comment:
A few points/nits:
* the continued lines would be easier to recognize if they were split before `&&`
* please omit the redundant quotes around words like "build" and "podman"
* please follow the convention used in the rest of this file which puts a space before `;` (like you did on line 262)